resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/k8s/integrationtest/VolcanoSuite.scala

```diff
@@ -25,6 +30,25 @@ class VolcanoSuite extends KubernetesSuite with VolcanoTestsSuite {
     sparkAppConf
       .set("spark.kubernetes.driver.scheduler.name", "volcano")
       .set("spark.kubernetes.executor.scheduler.name", "volcano")
+    testGroups = mutable.Set.empty
+  }
+
+  private def deletePodInTestGroup(): Unit = {
```

Comment:
FYI, These code also submited into a separate PR: https://github.com/apache/spark/pull/35733

Reply:
Got it. Thanks, @Yikun .

Reply:
I added my comment about `stackable` clean-up there.
- https://github.com/apache/spark/pull/35733#pullrequestreview-900547357

Reply:
@dongjoon-hyun addressed